### DISCUSSION TITLE
Do we have size limit of UserData?

### DISCUSSION THREAD

**Discussion by gou4shi1** [uNetworking/uWebSockets#1235] [Q&A]
If sizeof(MyUserData) <= 64, everything fine, else it crash...

**Comment by ghost** [comment]
No there is no limit. Can you give me an example where it crashes, and what version and configuration you use?

**Reply by gou4shi1** [reply]
It seem fixed in the latest version.

**Reply by gou4shi1** [reply]
I found some new options in the latest version.
```
/* This one depends on kernel timeouts and is a bad default */
bool resetIdleTimeoutOnSend = false;
```
why `false` is a bad default?

**Reply by hst-m** [reply]
its saying `true` is bad default, `false` = good

**Reply by gou4shi1** [reply]
now the default is `false`, so "is a bad default" is confused
